plugins/postcss-nesting/src/lib/merge-selectors/merge-selectors.ts

```diff
@@ -225,7 +227,7 @@ function nestingIsNotInsideCompoundSelector(selector) {
 	}
 
 	for (let i = 0; i < selector.parent.nodes.length; i++) {
-		if (!selector.parent.nodes[i].type === 'nesting') {
+		if (selector.parent.nodes[i].type === 'nesting') {
```

Comment:
typo? 🤔 
Not sure if this needs to be `===` or `!==`
But the `!` before `selector` was definitely incorrect.

Reply:
Good catch!